eth/watchers/senderwatcher.go

```diff
@@ -128,6 +134,11 @@ func (sw *SenderWatcher) Clear(addr ethcommon.Address) {
 	}
 }
 
+// SubscribeReserveChange notifies subscribers when the sender info for a particular sender changes
+func (sw *SenderWatcher) SubscribeReserveChange(sink chan<- ethcommon.Address) event.Subscription {
+	return sw.senderInfoScope.Track(sw.senderInfoFeed.Subscribe(sink))
```

Comment:
`reserveChangeScope`/`reserveChangeFeed` may be a better names than `senderInfoScope`/`senderInfoFeed` given how these fields are used.

Reply:
fixed in 66c5fa90